dashboard/src/main/java/com/google/cloud/tools/opensource/dashboard/DashboardArguments.java

```diff
@@ -101,6 +122,16 @@ private static Options configureOptions() {
             .build();
     inputGroup.addOption(inputCoordinatesOption);
 
+    Option versionlessCoordinatesOption =
```

Comment:
Instead of a separate option, would it be simpler to allow the version part of the coordinates of -c to be * or some other sentinel value?

-c "com.google.cloud:libraries-bom:*"

Reply:
I implemented one with version range https://github.com/GoogleCloudPlatform/cloud-opensource-java/pull/739 . Elliotte does not like it.